documentation/src/main/asciidoc/topics/proc_running_embedded.adoc

```diff
@@ -0,0 +1,54 @@
+[id='run_library']
+= Running {brandname} as an Embedded Library
+Learn how to run {brandname} as an embedded data store in your project.
+
+.Procedure
+
+* Initialize the default Cache Manager and add a cache definition as follows:
+
+[source,java,options="nowrap"]
+----
+import org.infinispan.Cache;
+import org.infinispan.commons.api.CacheContainerAdmin;
+import org.infinispan.configuration.cache.CacheMode;
+import org.infinispan.configuration.cache.Configuration;
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.configuration.global.GlobalConfigurationBuilder;
+import org.infinispan.manager.DefaultCacheManager;
+
+...
+
+      GlobalConfigurationBuilder global = GlobalConfigurationBuilder.defaultClusteredBuilder();
+            DefaultCacheManager cacheManager = new DefaultCacheManager(global.build());
+            ConfigurationBuilder builder = new ConfigurationBuilder();
+            builder.clustering().cacheMode(CacheMode.DIST_SYNC);
+      cacheManager.administration().getOrCreate("myCache", builder.build());
+----
+
+The preceding code initializes a default, clustered Cache Manager. Cache
+Managers contain your cache definitions and control cache lifecycles.
+
+{brandname} does not provide default cache definitions so after initializing
+the default Cache Manager, you need to add at least one cache instance. This
+example uses the `ConfigurationBuilder` class to create a cache definition that
+uses the distributed, synchronous cache mode. You then call the `getOrCreate()`
+method that either creates a cache named "myCache" or returns it if it already
+exists.
+
+.Next steps
+
+Now that you have a running Cache Manager with a cache created, you can add
+some more cache definitions, configure a cluster transport layer, put some data
```

Comment:
the transport must be configured before instantiating the `DefaultCacheManager`.